<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fscryptcreateoptions</class>
 <widget class="QDialog" name="fscryptcreateoptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Volume Creation Options </string>
  </property>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>101</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>101</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Ok</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Volume's Protector Name Below</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>421</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Key Type</string>
   </property>
   <widget class="QRadioButton" name="rbCustomKey">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>A Custom Passphrase</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbLoginPassPhrase">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Your Login Passphrase</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbRawKey">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>A raw 256-bit key</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
